app/workers/updates_synchronizer_worker.rb

```diff
@@ -7,15 +7,14 @@ def perform
     logger.info 'Running UpdatesSynchronizerWorker'
     logger.info 'Downloading...'
 
-    # TODO: this check can be removed when we switch to CDS data.
-    if TradeTariffBackend.use_cds?
+    if TradeTariffBackend.uk?
```

Comment:
uk? and use_cds? are the same. We could do with getting rid of the use_cds flag

Reply:
Yeh - lets go with that - we lose the ability to disable CDS sync but why we'd want to I don't know :man_shrugging:  and its another potential for misconfiguration

We can't disable Tariq in XI anyway